docs/CONTRIBUTING.md

```diff
@@ -8,6 +8,10 @@ The following is a set of guidelines for contributing to SecDevLabs. These are j
 
 Well, a good place to start is to make sure you can deploy your app with Docker-Compose. All the apps in SecDevLabs are built with it in order to ensure you can get get right into the action! ✓
 
+If you need help with it, make sure to check [Docker Compose Getting Started][6]. There you'll find examples for both `Dockerfile` and `docker-compose.yml`, two essential files for deploying your app.
```

Comment:
I think this paragraph should be part of the one above, as they are related. For me, it could look something like this:
`
All the apps in SecDevLabs are built with it in order to ensure you can get right into the action and, if you need help with it, make sure to check [Docker Compose Getting Started][6]. ✓
`

Reply:
I proposed another version in 84bf074. What do you think about it?

Reply:
That's perfect!